app/controllers/user_journeys_controller.rb

```diff
@@ -1,6 +1,10 @@
 class UserJourneysController < ApplicationController
   before_action :require_user
 
+  def index
+    @user_journeys = current_user.user_journeys.all.select{ |user_journey| !user_journey.all_steps_completed? && user_journey.started? }
```

Comment:
`.all` tu mozes vynechat

Reply:
Toto bude asi skareda iteracia co bude odpalovat vela selectov ale pre jedneho usera to asi nebude vela zacatych user_journeys cize mozno to mozme aj nechat tak.

Reply:
Toto bola/je moja najväčšia dilema na úlohe. Zvažovala som či to kontrolovať na úrovni databázy alebo takto škaredo. Nakoniec som to spravila takto jednoduchšie, nakoľko som brala do úvahy, že tých návodov v aplikácii nie je veľa, malo by to byť rýchle. Určite by si to žiadalo optimalizáciu pri väčšom objeme.